src/NServiceBus.Transport.Msmq/DelayedDelivery/DueDelayedMessagePoller.cs

```diff
@@ -298,7 +299,7 @@ async Task TrySendDelayedMessageToErrorQueue(DelayedMessage timeout, Exception e
                 Dictionary<string, string> headersAndProperties = MsmqUtilities.DeserializeMessageHeaders(timeout.Headers);
 
                 ExceptionHeaderHelper.SetExceptionHeaders(headersAndProperties, exception);
-
+                headersAndProperties[FaultsHeaderKeys.FailedQ] = timeoutsQueueTransportAddress;
```

Comment:
not sure how much sense that really makes given there is no failing queue in this case (as it's read from the persistence) but I've kept it to not change the current behavior. Not sure whether this might break something in SC for example.